apps/reflect/src/components/auth/AuthLoginButton.tsx

```diff
@@ -32,16 +35,11 @@ const AuthLoginButton: FC<IProps> = ({
                 borderRadius: "4px",
               },
             }
-          : { cursor: "not-allowed" }),
+          : { cursor: "not-allowed", color: "gray" }),
       }}>
-      {icon}
+      {<Icon {...iconProps} {...(disabled && { color: "gray" })} />}
```

Comment:
we realy need "..." on this props is it just take a value of true or false if i'm missing something please don't hesitate give a information 💯

Reply:
Yes we need it to add gray color when button is disabled. Also we need extra styles after on disabled styles